ClinicWeb/src/main/java/pl/infoshare/clinicweb/user/registration/RegistrationController.java

```diff
@@ -46,6 +46,35 @@ public String registerFormSubmission(@Valid @ModelAttribute("user") UserDto user
         return "redirect:/login";
     }
 
+    @GetMapping("/register-admin")
+    public String registerByAdmin(@ModelAttribute UserDto user, Model model) {
+
+
+        model.addAttribute("user", user);
+
+        log.info("New user registration form was requested.");
+
+        return "user/registry-admin";
+    }
+
+    @PostMapping("/register-admin")
+    public String userCreatedByAdmin(@Valid @ModelAttribute("user") UserDto user, BindingResult bindingResult,
```

Comment:
określanie w nazwie metody przez kogo został utworzony user jest zbędne
```suggestion
    public String createUser(@Valid @ModelAttribute("user") UserDto user, BindingResult bindingResult,
```